pkg/pulp_client/rpm_publication.go

```diff
@@ -40,3 +40,19 @@ func (r *pulpDaoImpl) FindRpmPublicationByVersion(ctx context.Context, versionHr
 		return nil, nil
 	}
 }
+
+func (r *pulpDaoImpl) DeleteRpmPublication(ctx context.Context, versionHref string) error {
```

Comment:
Deleting this manually is okay, but did you notice if pulp automatically deletes the publication when deleting the version?

Reply:
Good point, checked and you're right, the publication gets deleted automatically when deleting the version.
Removed this as it was unnecessary. 👍🏼